app/src/androidTest/java/tech/relaycorp/gateway/data/disk/AndroidPrivateKeyStoreTest.kt

```diff
@@ -8,37 +8,20 @@ import tech.relaycorp.awala.keystores.file.FileKeystoreRoot
 import tech.relaycorp.gateway.App
 import tech.relaycorp.relaynet.testing.pki.CDACertPath
 import tech.relaycorp.relaynet.testing.pki.KeyPairSet
-import tech.relaycorp.relaynet.testing.pki.PDACertPath
+import tech.relaycorp.relaynet.wrappers.privateAddress
 import java.io.File
 
 class AndroidPrivateKeyStoreTest {
     private val privateKey = KeyPairSet.PRIVATE_GW.private
-    private val certificate = PDACertPath.PRIVATE_GW
 
     @Test
     fun saveAndRetrieve() = runBlockingTest {
         val androidContext = ApplicationProvider.getApplicationContext<App>()
         val root = FileKeystoreRoot(File(androidContext.filesDir, "tmp-keystore"))
         val store = AndroidPrivateKeyStore(root, androidContext)
 
-        store.saveIdentityKey(privateKey, certificate)
-        val retrievedKeypair = store.retrieveIdentityKey(certificate.subjectPrivateAddress)
-        assertEquals(privateKey, retrievedKeypair.privateKey)
-        assertEquals(certificate, retrievedKeypair.certificate)
-    }
-
-    @Test
-    fun overrideKey() = runBlockingTest {
```

Comment:
IIRC, I had to add this test after finding out the hard way that I had to do things differently in the unit under test. If we remove it, we risk making the same mistake if we were to refactor the unit under test later.